install_virtualenv.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+virtualenv_path=$1
+
+root=`dirname $0`
+(cd ${root}/external/daint_venv && ./install.sh ${virtualenv_path})
```

Comment:
is this working with a commit to daint_venv? currently `install.sh` always goes into `/project[...]`, see https://github.com/VulcanClimateModeling/daint_venv/blob/c15c1d93700a644aac7f5d7f32889a02b9fb8d82/install.sh#L5

Reply:
yeah right now the daint_env submodule is pointing at a branch. I will open a parallel PR there so it can point to master

Reply:
ah, I did not see that. Makes sense